pset-functions.js

```diff
@@ -114,7 +151,10 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+const isPalindrome = word => word === word.split('').reverse().join('')
 
```

Comment:
Crushing it; this method chaining along with an arrow function that takes advantage of conditional operators returning a true/false value...bravo!